airbyte-config/config-models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -171,7 +171,36 @@ public interface Configs {
    */
   boolean runDatabaseMigrationOnStartup();
 
+  // Temporal Cloud
+
+  /**
+   * Define if Temporal Cloud should be used
```

Comment:
Should we add `internal use only` to all these java docs to make it clear these should not be appearing on public facing documentation?